### PR TITLE
Safenet

### PR DESCRIPTION
**Pull request by businesskaniki**
this is my first major project build with HTML CSS and js .it is a cyber security website.

### CODE REVIEW

File: safe.html [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+        </li>
+        <li class="top-li">
+          <a href="">mypage</a>
+        </li>
+        <li class="top-li">
+          <a href="">logout</a>
+        </li>
+      </ul>
+    </nav>
+    <!-- main navbar  -->
```
Comment (tufoinnkuo10) [inline safe.html]: - [ ] kindly make the mobile popup menu full height so its importance can be felt as we won't be able to navigate when it comes up unless we use it\
Screenshot (197)_LI

File: safe.html [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+      <div class="logdiv">
+        <img src="./images/logo.jpg" alt="logo" />
+        <a href="">safenet</a>
+      </div>
+      <ul class="nav-ul">
+        <li class="nav-li-active link"><a href="./about.html">About</a></li>
+        <li class="nav-li link"><a href="#">Program</a></li>
+        <li class="nav-li link"><a href="#">join</a></li>
+        <li class="nav-li link"><a href="#"> sponser</a></li>
+        <li class="nav-li link"><a href="#">News</a></li>
```
Comment (tufoinnkuo10) [inline safe.html]: - [ ] please add links to the different sections here so it makes it easier for a user to navigate

File: index.js [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+    title: 'reagan clinton',
+    titleDetail: 'senior software engeneer',
+    description: 'Prepares and installs solutions by determining and designing system specifications, standards, and programming',
+    imageSrc: './images/pexels-pixabay-354951.jpg',
+    imageAlt: 'first person',
+  },
+  {
+    title: 'reagan clinton',
+    titleDetail: 'senior software engeneer',
+    description: 'Prepares and installs solutions by determining and designing system specifications, standards, and programming',
```
Comment (tufoinnkuo10) [inline index.js]: - [ ] kindly make some changes here even if its just with the name so we know they are getting all the different objects from the array

File: safe.html [hunk 4]
```diff
@@ -0,0 +1,155 @@
...
+          </h3>
+          <p class="paragraph">
+           we also help companies to protect themselves from wireless or wide area networks attack
+          </p>
+        </div>
+      </div>
+      <div class="last-text">
+        <span>see all services</span>
+        <button type="button">join safenet today</button>
+      </div>
```
Comment (tufoinnkuo10) [inline safe.html]: - [ ] kindly use a brighter and more friendly color here.
Screenshot (198)_LI

File: safe.html [hunk 5]
```diff
@@ -0,0 +1,155 @@
...
+    <nav class="nav">
+      <div class="logdiv">
+        <img src="./images/logo.jpg" alt="logo" />
+        <a href="">safenet</a>
+      </div>
+      <ul class="nav-ul">
+        <li class="nav-li-active link"><a href="./about.html">About</a></li>
+        <li class="nav-li link"><a href="#">Program</a></li>
+        <li class="nav-li link"><a href="#">join</a></li>
+        <li class="nav-li link"><a href="#"> sponser</a></li>
```
Comment (tufoinnkuo10) [inline safe.html]: - [ ] sponsor nor sponser.. kindly fix that

**Review by tufoinnkuo10 — CHANGES_REQUESTED** [review]
Hi @businesskaniki 

Great job so far with the project. There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight
✅ You used the correct Github flow
✅ Descriptive pull request message
✅ linter error checks validated
✅ mobile version looks ok
✅ nice idea
✅ design well followed

## Required Changes ♻️♻️
Check the comments under the review
- [ required] please add a loom video of not more than five minutes showing your work and a feature you are proud to have worked on.
- [ required] kindly add an acknowledgment section to your readme and give appropriate credit to [Cindy Shin](https://www.behance.net/adagio07) the [author of the original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), as required by the [Creative Commons License](https://creativecommons.org/licenses/).
- [ required] please make use of live pages and deploy your work also adding a link of it to the readme file
- [required] add a good description of what you did/build
- [ required] add a good and brief pull request description

tufoin

Optional suggestions
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

File: safe.html (hunk 1 above)
Comment (KDlamini) [reply safe.html]: @businesskaniki this requirement is not fulfilled.

File: index.html [hunk 6]
```diff
@@ -0,0 +1,154 @@
...
+        <img src="./images/logo.jpg" alt="logo" />
+        <a href="">safenet</a>
+      </div>
+      <ul class="nav-ul">
+        <li class="nav-li-active link"><a href="./about.html">About</a></li>
+        <li class="nav-li link"><a href="#prog">Program</a></li>
+        <li class="nav-li link"><a href="#">join</a></li>
+        <li class="nav-li link"><a href="#"> sponsor</a></li>
+        <li class="nav-li link"><a href="#">News</a></li>
+      </ul>
```
Comment (KDlamini) [inline index.html:50]: On mobile view, kindly make sure that the mobile menu fills the entire height of the page as requested by the previous reviewer. I should not be able to see the background content when the menu is active.

image

- Kindly consider setting the height to 100vh on the active menu.

**Review by KDlamini — CHANGES_REQUESTED** [review]
### Hi @businesskaniki  🙋‍♀️,  

You have done an incredible job implementing some of the changes requested by the previous reviewer. 👏🏾👏🏾👏🏾  
However, there are still some changes that aren't addressed yet. You are almost there! 💪🏾 

## Required Changes ♻️
 
**Kindly look through the comments below 👇 for required changes**

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

N/A

Cheers 🥂  and Happy coding!!! 👯 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @KDlamini in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by omarsalem7 — APPROVED** [review]
Hi @businesskaniki 😊, 
# STATUS: APPROVED! 🥇 💯 

giphy

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 🚀 
Congratulations! 🎉 

## Highlights
✅ No linter errors. 
✅ Descriptive pull request 
✅ Project has met all the technical aspects you needed for this project. Keep rocking! 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**